demo/api/src/products/product.crud.resolver.ts

```diff
@@ -1,28 +1,26 @@
 // Scaffolded by the CRUD generator on 2023-03-20.
```

Comment:
Out of scope: Should we remove products altogether? Is this needed for the module?

Reply:
I don't think it's needed (@raphaelblum, @RainbowBunchie ?). So yes, we could remove it completely. But that's not urgent

Reply:
They are not needed, so they can be removed in the future